SwagMediaSftp.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/*
+ * (c) shopware AG <info@shopware.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ */
+
+namespace SwagMediaSftp;
+
+require __DIR__ . "/vendor/autoload.php";
+
+use Shopware\Components\Plugin;
+use Shopware\Components\Plugin\Context\ActivateContext;
+use Shopware\Components\Plugin\Context\DeactivateContext;
+use Shopware\Components\Plugin\Context\InstallContext;
+use Shopware\Components\Plugin\Context\UninstallContext;
+use Shopware\Components\Plugin\Context\UpdateContext;
+
+class SwagMediaSftp extends Plugin
+{
+    public function update(UpdateContext $context)
```

Comment:
the overwriting of these four methods is completely unnecessary, as they do exakt the same as the parent methods

Reply:
Indeed, I didn't check that. Removed the methods